packages/recs/src/v2/abiTypesToRecsTypes.ts

```diff
@@ -0,0 +1,203 @@
+import { Type as RecsType } from "../constants";
+import { AbiType } from "@latticexyz/schema-type";
+
+export const abiTypesToRecsTypes = {
+  uint8: RecsType.Number,
+  uint16: RecsType.Number,
+  uint24: RecsType.Number,
+  uint32: RecsType.Number,
+  uint40: RecsType.Number,
+  uint48: RecsType.Number,
+  uint56: RecsType.Number,
+  uint64: RecsType.Number,
+  uint72: RecsType.Number,
+  uint80: RecsType.Number,
+  uint88: RecsType.Number,
```

Comment:
Is anything above `uint53` really represented as `number` in `recs`? Since those don't fit into javascript `number` and `recs` in its current state doesn't support `bigint`, I would expect them to be represented as a hex string (= `RecsType.String`)

Reply:
Defer to @holic (this isn't really a part of this PR, as I just refactored the previous mapping of schema types)

Reply:
does RECS consistently give us back a native number? iirc some things use `BigNumber` under the hood